concourse/scripts/common.sh

```
@@ -1303,345 +1116,116 @@ function wait_dss_until_ready(){
-        kill $pid
-      fi
-    done
-
-    wait_until_finished
+# Function to wait until data store server is finished
+function wait_dss_until_finished() {
+  local interval=1
+  local timeout=120
+  local elapsed=0
 
+  while [ $(ps aux | grep dss_server | grep -v grep | wc -l) -gt 0 ]; do
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)